platform/security/certificate/security-certificate-generator/src/main/java/org/codice/ddf/security/certificate/generator/CertificateCommand.java

```diff
@@ -28,66 +30,102 @@
 
     /**
      * Pass in a string to use as the common name of the certificate to be generated.
-     * Exception thrown if 0 arguments or more than 1 argument.
+     * Exception thrown if 0 arguments or more than 2 argument (or 4 when -san is used).
      *
      * @param args
      */
     public static void main(String args[]) {
-        if (args.length != 2) {
+        // try to extract -san if provided
+        int expected = 2;
+        String[] sans = null;
+        int mainIndex = 0;
+
+        if (args.length == 4) {
+            if (args[0].trim()
+                    .equalsIgnoreCase("-san")) {
+                sans = Arrays.stream(args[1].split("[,]"))
+                        .map(String::trim)
+                        .toArray(String[]::new);
+                mainIndex = 2;
+                expected = 4;
+            } else if (args[2].trim()
+                    .equalsIgnoreCase("-san")) {
+                sans = Arrays.stream(args[3].split("[,]"))
+                        .map(String::trim)
+                        .toArray(String[]::new);
+                expected = 4;
```

Comment:
I think it is meant to be called from the command line window before you start DDF. It could be simplified with apache command but I didn't want to introduce it here just for that. I would say we should upgrade to the apache library if we ever need to add more arguments than that.